### DISCUSSION TITLE
Some achievements don't even make sense

### DISCUSSION THREAD

**Discussion by jabezborja** [community/community#27464] [Other Feature Feedback, Questions, & Ideas]
Please make achievements that employers could use as a benchmark for their abilities.

**Comment by jfullstackdev** [comment]
what are those if you can give ideas ? they may be implemented in the next release of new badges

**Reply by gabrielgaraujo** [reply]
I've unlocked the achievement [Quick Draw](https://github.com/users/gabrielgaraujo/achievements/quickdraw)

But I was the one opening the PR by mistake and closing it within 5 minutes. Not sure if that was the intended scenario for this achievement. Maybe it should be reviewed and only unlock when someone else merges your PR instead of just closing the PR.

**Reply by jfullstackdev** [reply]
I believe the idea behind is to close a PR within 5 minutes in a collaborative scenario, which can mean the team is productive enough

**Reply by gabrielgaraujo** [reply]
Yes, I think that was the original intention for this achievement, so the scenario where I got it feels like cheating.
How can we report this so it can be reviewed and adjusted to feel more rewarding rather than something anyone can fabricate?
